doc/cookbook/source/examples/kernel/custom_kernel.rst

```diff
@@ -0,0 +1,37 @@
+=============
+Custom Kernel
+=============
+
+The Custom Kernel allows for custom user provided kernel matrices.
+
+-------
+Example
+-------
+In this example, we initialize a :sgclass:`CGaussianKernel` and then use the kernel matrix to create a custom kernel.
+
+Imagine we have files with data.
+We create :sgclass:`CDenseFeatures` (here 64 bit floats aka RealFeatures) as
+
+.. sgexample:: custom_kernel_machine:create_features
+
+We create an instance of :sgclass:`CGaussianKernel`.
+We initialize with :sgclass:`CDenseFeatures` and then get the kernel matrix.
+
+.. sgexample:: custom_kernel_machine:create_kernel
+
+We use the provided kernel matrix to create custom kernels.
+
+.. sgexample:: custom_kernel_machine:create_custom_kernel
+
+We create an instance of :sgclass:`CLibSVM` with the custom kernel.
+
+.. sgexample:: custom_kernel_machine:create_machine
+
+Then we train the :sgclass:`CLibSVM` and we apply it to the test data, which gives the predicted :sgclass:`CBinaryLabels`.
```

Comment:
i guess we dont talk about specialized label classes anymore, just CLabels, but that is totally minor